test/e2e-go/e2e_go_suite_test.go

```diff
@@ -59,43 +52,17 @@ var _ = BeforeSuite(func(done Done) {
 	By("creating a new directory")
 	Expect(tc.Prepare()).To(Succeed())
 
-	By("checking the cluster type")
-	kubectx, err = tc.Kubectl.Command("config", "current-context")
+	By("getting the cluster Kind")
```

Comment:
i dont understand this change (or the original wording, for that matter). Isn't this getting the current context (presumably to talk to the cluster)?

Reply:
I am ok with change that name to `getting the current context` it is fine.